tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 skipsdist = true
 envlist =
+    intro,
```

Comment:
intro is overloaded with the first spam notebook, can we come up with a consistent different name for them?

Reply:
Okay changing to `getting_started`